docs/source/scheduler.rst

```diff
@@ -0,0 +1,10 @@
+.. _Scheduler:
+
+=========
+SCHEDULER
+=========
+
+MLflow tracking server provides a way to add periodic tasks using flask-apscheduler.
+To activate the scheduler, you need to provide a configuration file to the command line that launches
```

Comment:
Suggestion: simplify the sentence.

To activate the scheduler: provide the path to the configuration file with the command line to launch using `--scheduler-configuration` option